.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           name: deploy-to-serverless
           command: |
             sls package
```

Comment:
since this is being run in a shell, you could put a line
```
export CODE_VERSION=${CIRCLE_SHA1:0:7}
```
and then set the env var to this in the serverless.yml file